stdlib/sys.pyi

```diff
@@ -359,3 +359,10 @@ if sys.version_info < (3, 8):
 # as part of the response to CVE-2020-10735
 def set_int_max_str_digits(maxdigits: int) -> None: ...
 def get_int_max_str_digits() -> int: ...
+
+if sys.version_info >= (3, 12):
+    def getunicodeinternedsize() -> int: ...
+    if sys.platform == "linux":
+        def activate_stack_trampoline(__backend: str) -> None: ...
+        def deactivate_stack_trampoline() -> None: ...
+        def is_stack_trampoline_active() -> bool: ...
```

Comment:
According to stubtest, these exist on all platforms, not just Linux

Reply:
Ok, here's what is going on:

```python
>>> import sys
>>> sys.activate_stack_trampoline('perf')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: perf trampoline not available
```

Source: https://github.com/python/cpython/blob/3.12/Python/sysmodule.c#L2166-L2168

I think that I can add a `def activate_stack_trampoline(__backend: str) -> NoReturn` for non-linux versions.

Reply:
I'd actually be okay with keeping the sys.version_info branch and adding allowlist entries on Windows/Darwin in that case! (We should add a comment to the allowlist saying why they're not included, though).

Reply:
I think that adding it is better, users might do something like:

```python
try:
   sys.activate_stack_trampoline('perf')
except ValueError:
   some_other_case()
```

And it is quite easy to do with conditional definition.

Reply:
Yeah, makes sense!